src/components/Display.js

```diff
@@ -0,0 +1,7 @@
+export default function Display (props){
```

Comment:
use object destruction

```suggestion
export default function Display ({displayValue}){
```